### ISSUE TITLE
Add images of the Raspberry Pi board

### ISSUE THREAD

**Issue by gohai** [processing/processing-pi-website#9]
It occurred to me that a prominently placed image of the Raspberry Pi board, potentially in combination with "Processing" (PDE?), could help people understand what this page is about... @msurguy ideas?

**Comment by msurguy** [comment]
@gohai yes, I think that's a great idea to have some graphic put together!

I've taken some photos and screenshots of the PDE running on the Pi 3 and on Pi Zero. 

Here is the Pi 3 PNG file that I took with my camera so that we don't have to worry about licensing issues. After that photo, the screenshots of the PDE follow.  

![pi3b](https://user-images.githubusercontent.com/585833/41579387-a8da5222-734b-11e8-90eb-8a99700ac2de.png)


![pi-processing-launch-screen](https://user-images.githubusercontent.com/585833/41579377-a1075e1e-734b-11e8-91b8-0e70056c1cd7.jpg)


![pi-processing-pulses](https://user-images.githubusercontent.com/585833/41579370-9a0f28ee-734b-11e8-9ddb-2ade33bc02e0.jpg)

**Comment by msurguy** [comment]
@gohai just to be sure, you are talking about the homepage? (https://pi.processing.org/)

What kind of graphic would you like me to make? I can make a photo of a little HDMI screen, crop just the screen and combine with the photo of the board or I am open to suggestions!

**Comment by msurguy** [comment]
@gohai Here's a simple graphic I made that might be useful:

![processingwithpi](https://user-images.githubusercontent.com/585833/41744706-772fc03a-7559-11e8-811b-1eee1899869a.jpg)

Let me know if you want something else
